### ISSUE TITLE
Rate Limiting

### ISSUE THREAD

**Issue by maxim123** [mterwill/office-hours-help-queue#102]
If ≥ `n` students are signed up on a queue, prevent students from creating a new request during the first `t` minutes after their previous request has been resolved, so as to prevent students from signing up immediately after they have been helped. Let the course's staff set `n` and `t` in the Admin Panel.

**Comment by mterwill** [comment]
I don't think this would be too bad. We could add new columns for `n` and `t` on the `Course` model, along with getters/setters in the admin interface. Then just create a custom validator on `CourseQueueEntry`, which already stores `requester_id` and a `resolved_at` timestamp.

One catch is that there's not currently any error handling on the frontend, so some functionality would need to be added to propagate a useful message up to the end user letting them know why their enqueue was rejected.
